action.yml

```diff
@@ -1,8 +1,12 @@
-name: 'Wait'
+name: 'Constable'
 description: 'Create a JavaScript Action grade the project'
+inputs:
+
 outputs:
-  grade:
-    description: 'The output of this file will be a grade and percentage given to the repo.'
+  score:
```

Comment:
I thought we wanted `grade`, `score`, and `report` based on our issue conversation. 

https://github.com/dangoslen/constable-github-action/issues/4

Reply:
So I am keeping the following outputs:
1. Score
2. Grade
3. Report
Hope this is what you guys are expecting.